File: php/AKsesoris.php
<?php
require_once "PetShop.php";

class Aksesoris extends PetShop {
    protected $jenis;
    protected $bahan;
    protected $warna;

    public function __construct($id, $nama_produk, $harga_produk, $stok_produk, $foto_produk, $jenis, $bahan, $warna) {
        parent::__construct($id, $nama_produk, $harga_produk, $stok_produk, $foto_produk);
        $this->jenis = $jenis;
        $this->bahan = $bahan;
        $this->warna = $warna;
    }

    public function getDetail() {
        return parent::getDetail() . "
            <p>Jenis: {$this->jenis}</p>
            <p>Bahan: {$this->bahan}</p>
            <p>Warna: {$this->warna}</p>
            <hr>
        ";
    }
}
?>

File: CPP/AKsesoris.cpp
#include "PetShop.cpp"

class Aksesoris : public PetShop {
protected:
    string jenis;
    string bahan;
    string warna;

public:
    Aksesoris(int id, string nama_produk, int harga_produk, int stok_produk, string foto_produk,
          string jenis, string bahan, string warna)
        : PetShop(id, nama_produk, harga_produk, stok_produk, foto_produk), jenis(jenis), bahan(bahan), warna(warna) {}

    void displayAksesoris() {
        displayPetShop();
        cout << "Jenis: " << jenis << endl;
        cout << "Bahan: " << bahan << endl;
        cout << "Warna: " << warna << endl;
    }
};
